django_tasks/checks.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any, List, Sequence
+
+from django.apps.config import AppConfig
+from django.core.checks.messages import CheckMessage
+
+from django_tasks import tasks
+
+
+def check_tasks(
+    app_configs: Sequence[AppConfig] = None, **kwargs: Any
+) -> List[CheckMessage]:
+    """Checks all registered task backends."""
+
+    errors = []
+    for backend in tasks.all():
+        try:
+            backend_errors = backend.check()
+        except NotImplementedError:
+            pass
```

Comment:
Issue: This could hide issues, so we should let this raise up, and noop the `.check` instead.